test/common_factory_test.h

```diff
@@ -51,20 +58,73 @@ class CommonFactoryTest : public ::testing::Test
         ZoneId* zone_id = ZoneId::default_instance().New();
         zone_id->set_value(ZONE_ID);
         zone_offset->set_allocated_id(zone_id);
-        zone_offset->set_amountseconds(42);
+        zone_offset->set_amount_seconds(42);
 
-        params.set_actor(actor)
+        params_.set_actor(actor)
                 .set_tenant_id(tenant_id)
                 .set_zone_offset(zone_offset);
 
-        query_factory_ = ActorRequestFactory::create(params).query_factory();
-        command_factory_ = ActorRequestFactory::create(params).command_factory();
-        topic_factory_ = ActorRequestFactory::create(params).topic_factory();
+        query_factory_ = ActorRequestFactory::create(params_).query_factory();
+        command_factory_ = ActorRequestFactory::create(params_).command_factory();
+        topic_factory_ = ActorRequestFactory::create(params_).topic_factory();
+    }
+
+    std::vector<std::unique_ptr<ProjectId>> make_project_ids()
+    {
+        std::vector<std::unique_ptr<ProjectId>> project_ids;
```

Comment:
Could be refactor 
std::vector<std::unique_ptr<ProjectId>> project_ids;
for(inti  = 0; i < 4; i++) {
auto project_id = std::make_unique_ptr<ProjectId>());
project_id->set_value(i);
project_ids.emplace_back(project_id);
}